doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -322,8 +323,15 @@ Cellular samples
 
 * :ref:`nrf_cloud_multi_service` sample:
 
-  * Updated Wi-Fi overlays from newlibc to picolib.
-  * Added the :kconfig:option:`CONFIG_TEST_COUNTER_MULTIPLIER` Kconfig option to multiply the number of test counter messages sent, for testing purposes.
+  * Added:
+
+    * The :kconfig:option:`CONFIG_TEST_COUNTER_MULTIPLIER` Kconfig option to multiply the number of test counter messages sent, for testing purposes.
+    * A handler for new nRF Cloud event type ``NRF_CLOUD_EVT_RX_DATA_DISCON`` which stops sensors and location services.
```

Comment:
```suggestion
    * A handler for new nRF Cloud event type ``NRF_CLOUD_EVT_RX_DATA_DISCON`` to stop sensors and location services.
```
To remove the possibility of this being read as the event stopping sensors or services instead of the handler.